### PR TITLE
doc(readme): pallet-move reference info

### PR DESCRIPTION
**Pull request by Rqnsom**
Some doc updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,225 +1,7 @@
-# Substrate Node Template
+# Substrate Node Template With The Move Pallet Integrated
 
-A fresh [Substrate](https://substrate.io/) node, ready for hacking :rocket:
+This is just a fork of the Substrate Node Template (from the main branch of this repo) which integrates [pallet-move] inside and should only be used for testing.
```
Comment (neutrinoks) [inline README.md]:
```suggestion
This repository is just a fork of the Substrate Node Template (from the main branch of this repo), which integrates [pallet-move](https://github.com/eigerco/pallet-move) inside and should only be used for testing.
```

File: README.md [hunk 2]
```diff
@@ -1,225 +1,7 @@
...
 
-A standalone version of this template is available for each release of Polkadot
-in the [Substrate Developer Hub Parachain
-Template](https://github.com/substrate-developer-hub/substrate-parachain-template/)
-repository. The parachain template is generated directly at each Polkadot
-release branch from the [Node Template in
-Substrate](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/bin/node-template)
-upstream
+Check [pallet-move] if you would like to know more - even the _how-to-run_ instructions are located in that repo.
```
Comment (neutrinoks) [inline README.md:5]:
```suggestion
Check [pallet-move](https://github.com/eigerco/pallet-move)  if you would like to know more - even the _how-to-run_ instructions are located in that repo.
```

**Review by neutrinoks — CHANGES_REQUESTED** [review]
Looks good :+1: only little details such as URLs to be added

File: README.md (hunk 1 above)
Comment (Rqnsom) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Rqnsom) [reply README.md:5]: Done

**Review by neutrinoks — APPROVED** [review]

**Pull request merged**